src/kirby/components/segmented-control/segmented-control.component.tns.html

```diff
@@ -0,0 +1,15 @@
+  <StackLayout orientation="horizontal" horizontalAlignment="left" class="segment-control">
+    <ng-container *ngFor="let item of segmentItems">
+      <GridLayout columns="auto, auto" (tap)="onSegmentTap(item)">
+        <ng-container *ngIf="item.badge">
+          <FlexBoxLayout class="segment-badge-wrapper" col="1" flexDirection="row" alignItems="center" justifyContent="flex-end">
+            <kirby-badge text="{{item.badge.content}}" colortype="{{item.badge.type}}"></kirby-badge>
+          </FlexBoxLayout>
+        </ng-container>
+        <StackLayout class="segment-item" col="0" colSpan="2" verticalAlignment="center" [ngClass]="{'is-checked': item.checked}">
+          <Label text="{{item.text}}" value="{{item.id}}"></Label>
```

Comment:
Consider using property bindings, eg. `[text]="item.text"` instead of text-interpolation, since no text is actually being interpolated.